src/events/cd-event-list-item.vue

```diff
@@ -36,6 +32,14 @@
         {{ isFull ? $t('Full') : $t('See Details and Book') }}
       </router-link>
     </div>
+    <div v-if="isRecurring" class="cd-event-list-item__recurring-info">
+      <span class="fa fa-info-circle cd-event-list-item__recurring-info-icon"></span>
+      <span class="cd-event-list-item__recurring-info-header">{{ $t('This is a recurring event') }}</span>
+      <div class="cd-event-list-item__recurring-info-text">
+        {{ `${recurringFrequencyInfo} ${$t('at')}` }} {{ event.dates[0].startTime | cdTimeFormatter }} - {{ event.dates[0].endTime | cdTimeFormatter }},
```

Comment:
It's an edge case that happened at least once. My guess is it'll work 98% of the time, but I've seen in the past events with the latest date not being the latest. Could have been a temp glitch too, but considering we're manipulating an array of dates, i'd wouldn't rely on the index to order them.